### DISCUSSION TITLE
Russian users

### DISCUSSION THREAD

**Discussion by didiermcleod** [ciderapp/Cider#747] [General]
Unfortunately, Russian users YET cannot thank the creators of such a wonderful application. Microsoft forbids Russians from buying anything, even those who do not support the military operation in Ukraine.

Therefore, for now, I can only say my sincere thanks.

**Comment by KillerParticles** [comment]
Unless I'm missing something, you don't need to buy Cider to use it?

**Reply by didiermcleod** [reply]
<a href="https://www.imagebam.com/view/ME90CUQ" target="_blank"><img src="https://thumbs4.imagebam.com/0f/08/19/ME90CUQ_t.png" alt="Снимок экрана 2022-03-30 190544.png"/></a>

**Comment by tva2000hn** [comment]
I think you can still support the app via Open Collective, they support crypto AFAIK

**Comment by KillerParticles** [comment]
I remember seeing Crypto as an option for either OpenCollective or Ko-fi.

**Comment by jannuary** [comment]
Ой, случилось что-то?
